Pat/BreedList/BreedListViewController.swift

```diff
@@ -15,16 +16,36 @@ class BreedListViewController: UIViewController {
             }
         }
     }
+
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        updateFavoutitesList()
         dataSource.configureCell = { item, cell in
-            cell.textLabel?.text = item.name
+            let breed = item.name
+            cell.textLabel?.text = breed
+            cell.setButtonInitialImage(breed: breed)
+            cell.updateFaveList = {
+                if self.favourites.isFavourite(breed: breed) {
+                    self.favourites.removeBreed(breed: breed)
+                } else {
+                    self.favourites.addBreed(breed: breed)
+                }
+            }
         }
+
+        tableView.register(BreedCell.self, forCellReuseIdentifier: "ReusableCell")
         tableView.dataSource = dataSource
         tableView.delegate = self
         loadBreedList()
     }
 
+    func updateFavoutitesList() {
+        if let list = favourites.defaults.object(forKey: "favouritesListKey") as? [String] {
```

Comment:
vc tá dando update na lista usando os dados da lista? pq tá acessando o user defaults de fora?

Isso é só pra carregar a lista inicialmente? Se for, acho que pode ser feito dentro da classe de Favorites

Reply:
É só pra carregar inicialmente. Verdade, dá pra fazer dentro da favorites